src/core/primitives/RovingFocusGroup/RovingFocusGroup.tsx

```
@@ -0,0 +1,32 @@
+export interface RovingFocusGroupProps {
+    items: RovingFocusGroupItems[];
+  }
+
+
+function RovingFocusGroup({items}: RovingFocusGroupProps ) {
+    
+    return (
+        <RovingFocusRoot>
+            {items.map((item) => {
+                return (
+                    <RovingFocusItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)